cmake/kokkoskernels_tribits.cmake

```diff
@@ -10,7 +10,7 @@ MESSAGE(STATUS "The project name is: ${PROJECT_NAME}")
 MACRO(KOKKOSKERNELS_PACKAGE_POSTPROCESS)
   IF (KOKKOSKERNELS_HAS_TRILINOS)
     TRIBITS_PACKAGE_POSTPROCESS()
-  ELSEIF(NOT KOKKOSKERNELS_HAS_PARENT)
+  ELSE()
```

Comment:
These changes are not the correct behavior. Even if you are building Kokkos Kernels in-tree, you would still want to install its package file.